src/Http/routes.php

```diff
@@ -24,3 +24,7 @@
 
     Route::post('/vote/polls/{poll}', 'VoteManagerController@vote')->name('poll.vote');
 });
+
+Route::group(['namespace' => 'Inani\Larapoll\Http\Controllers', 'middleware' => 'web'], function() {
```

Comment:
no need to create new group, it should be included in the parent group
the route itself should be as the others.

Reply:
I want to avoid the prefix of the dashboard during viewing a single poll publically

Reply:
I don't really understand, there is already a method to display a single poll publically.

Reply:
I agree with you but it uses the prefix from config
`'prefix' => env('LARAPOLL_PREFIX', 'admin_polls'),`

And I think it doesn't make sense to show the prefix `admin_poll` to voters.

Reply:
The show method doesn't need to be under any specific routes. I invite you to take a look on the blog post I wrote about it.

Reply:
I agree with you, I saw the blog post.
But the issue is the polls listing doesn't show the links to show created polls individually.